css/style.css

```diff
@@ -537,14 +538,19 @@ ul.chars{
 }
 
 .feature-image {
-    max-height: 300px;
-    width: auto;
-    max-width: 500px;
+    height: 100%;
+    width: 100%;
+    max-height: 16em;
+    object-fit: contain;
+    display: block;
+    margin-left: auto;
+    margin-right: auto;
+    
 }
 
 .feature-paragraph {
     margin-left: 0px;
-    text-align: left;
+    text-align: center;
```

Comment:
I think center-aligning this looks better.

Reply:
center aligned text is very difficult to read for some people (like myself) and if you expect it to wrap more than one time you probably shouldn’t use it

Reply:
That class was misnamed, it's only used on the title text for the feature page. I'll rename it in my next commit.

Reply:
i dont think this should be done, the left align looks better imo